src/settings.cpp

```diff
@@ -2081,19 +2093,24 @@ static void Load_PATS()
 	/* Copy over default setting since some might not get loaded in
 	 * a networking environment. This ensures for example that the local
 	 * currency setting stays when joining a network-server */
-	LoadSettings(_settings, &_settings_game);
+	LoadSettings(_settings, &_settings_game, _settings_sl_compat);
 }
 
 static void Check_PATS()
 {
-	LoadSettings(_settings, &_load_check_data.settings);
+	LoadSettings(_settings, &_load_check_data.settings, _settings_sl_compat);
 }
 
-static void Save_PATS()
+static void RealSave_PATS(void *unused)
```

Comment:
There are compilers which would warn "'unused' is unused", which can be avoided by not naming the parameter.
```
static void RealSave_PATS(void *)
```